schemas/2015-10-31/Microsoft.Automation.json

```diff
@@ -147,70 +138,6 @@
       ],
       "description": "Microsoft.Automation/automationAccounts/certificates"
     },
-    "automationAccounts_compilationjobs": {
```

Comment:
>automationAccounts_compilationjobs [](start = 5, length = 34)

removing these is a breaking change. why do you remove them?

Reply:
Understood. Fixed, and sent a new commit.